### ISSUE TITLE
How to validate `compose` from other libraries?

### ISSUE THREAD

**Issue by ipanasenko** [jfmengels/eslint-plugin-lodash-fp#57]
Hi, guys.

I'm using [recompose](https://github.com/acdlite/recompose/) library in my React project, and I'd like this plugin to work for its `compose` method. The rule I'm particularly interested in is `no-single-composition`.

Is there a way to make this plugin work with it?

Code example that doesn't fail, but should:

```js
import { compose, withProps } from 'recompose';

export const SomeDiv = compose(
  withProps({ extraProp: 'bla-bla' })
)(() => <div>just some div</div>);
```
